react-task/src/App.js

```diff
@@ -1,25 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import Register from "./components/Register";
```

Comment:
We are generally cognisant of our import ordering as it makes imports easier to locate. So I would follow this structure:

1. Vendor imports (external libraries like React etc)
2. Custom components/files
3. CSS files

All separated by a blank line

```suggestion
import Register from "./components/Register";

import "./App.css";
```